examples/00-Landscaper-Configuration.yaml

```diff
@@ -5,6 +5,15 @@ repositoryContext: # define the default repository context for installations
   type: ociRegistry
   baseUrl: "myregistry.com/components"
 
+controllers:
+  context:
+    config:
+      disableDefault: false
+      repositoryContext: # define the default repository context for installations
+        type: ociRegistry
+        baseUrl: "myregistry.com/components"
+
```

Comment:
I'm a little bit confused here.
Is it

```yaml
controllers:
  context:
    config:
      default:
        disable: false
```

or

```yaml
controllers:
  context:
    config:
      disableDefault: false
```

Are both schemes allowed?

Reply:
missed that one during some refactoring.
the first one is correct